app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  validates :name, presence: true,
+    uniqueness: true,
+    format: { with: /\A[a-z][a-z\s]+[a-z]\z/,
+    message: "only allows underscore letters and whitespace" }
```

Comment:
My mistake, you are right it should be lowercase letters. We need string only with letters, as ingredients do not have numbers. 
Regarding second question. The way it is said now with the format, it covers the min. no. of letters, taking into account whitespace and lowercase letters, so I think going with validation against presence would make the code more complicated and longer. However I will leave the decision to you -> mentors.